nilearn/_utils/niimg_conversions.py

```diff
@@ -265,7 +265,7 @@ def concat_niimgs(niimgs, dtype=np.float32, accept_4d=False,
         else:
             nii_str = "image #" + str(index)
         if verbose > 0:
-            print "Concatenating {}/{}: {}".format(index + 1, sum(lengths),
+            print "Concatenating {0}/{1}: {2}".format(index + 1, sum(lengths),
                                                    nii_str)
```

Comment:
bonus points if you fix this PEP8 warning, while you are touching the line above.